src/langs/zh_tw.js

```diff
@@ -10,48 +10,63 @@
 
 // jshint camelcase:false
 jQuery.trumbowyg.langs.zh_tw = {
-    viewHTML: '原始碼',
-
-    formatting: '格式',
-    p: '段落',
-    blockquote: '引用',
-    code: '代碼',
-    header: '標題',
-
-    bold: '加粗',
-    italic: '斜體',
-    strikethrough: '刪除線',
-    underline: '底線',
-
-    strong: '加粗',
-    em: '斜體',
-    del: '刪除線',
-
-    unorderedList: '無序列表',
-    orderedList: '有序列表',
-
-    insertImage: '插入圖片',
-    insertVideo: '插入影片',
-    link: '超連結',
-    createLink: '插入連結',
-    unlink: '取消連結',
-
-    justifyLeft: '靠左對齊',
-    justifyCenter: '置中對齊',
-    justifyRight: '靠右對齊',
-    justifyFull: '左右對齊',
-
-    horizontalRule: '插入分隔線',
-
-    fullscreen: '全螢幕',
-
-    close: '關閉',
-
-    submit: '確定',
-    reset: '取消',
-
-    required: '必需的',
-    description: '描述',
-    title: '標題',
-    text: '文字'
-};
+    viewHTML: "原始碼",
+    formatting: "格式",
+    p: "段落",
+    blockquote: "引用",
+    code: "代碼",
+    header: "標題",
+    bold: "加粗",
+    italic: "斜體",
+    strikethrough: "刪除線",
+    underline: "底線",
+    strong: "加粗",
+    em: "斜體",
+    del: "刪除線",
+    unorderedList: "無序列表",
+    orderedList: "有序列表",
+    insertImage: "插入圖片",
+    insertVideo: "插入影片",
+    link: "超連結",
+    createLink: "插入連結",
+    unlink: "取消連結",
+    justifyLeft: "靠左對齊",
+    justifyCenter: "置中對齊",
+    justifyRight: "靠右對齊",
+    justifyFull: "左右對齊",
+    horizontalRule: "插入分隔線",
+    fullscreen: "全螢幕",
+    close: "關閉",
+    submit: "確定",
+    reset: "取消",
+    required: "必需的",
+    description: "描述",
+    title: "標題",
+    text: "文字",
+    noembed: "影片",
+    file: '檔案',
+    width: '寬',
+    base64: "base64圖片",
+    lineheight: '間距大小',
+    lineheights: {
+        '0.9': '間距小',
+        'normal': '預設間距',
+        '1.5': '間距中',
+        '2.0': '間距大'
+    },
+    fontFamily: '字體',
+    fontsize: '字體大小',
+    fontsizes: {
```

Comment:
Do not edit this file.

Translations for plugins should be **IN** the plugin file since they are useless for the core.